pkg/controller/controller.go

```diff
@@ -131,6 +148,22 @@ func NewResizeController(
 	return ctrl
 }
 
+func (ctrl *resizeController) initUncertainPVCs() error {
```

Comment:
please also handle the pvc changes on vac in the `updatePVC` event handler. According to current implementation, it will ingore the update event so that we have to wait at least 10 minutes for the pv to be modified or recieve a resize event.